### PR TITLE
bgpq4: init at 0.0.6

### PR DESCRIPTION
**Pull request by vincentbernat**
###### Motivation for this change

This is a fork of bgpq3. It is still unclear if it will superseed
bgpq3. It is mostly compatible with additional features, like support
for Mikrotik devices. It also defaults to NTT IRR server instead of
RADB.

cc @B4dM4n 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Lassulus** [comment]
hi, can you split the commit into 2? one for the maintainer addition and one for the package.
Also sorry for the long delay

**Comment by vincentbernat** [comment]
Sure! Done.

File: pkgs/tools/networking/bgpq4/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "bgpq4";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    owner = "bgp";
+    repo = pname;
+    rev = "${version}";
```
Comment (Lassulus) [inline pkgs/tools/networking/bgpq4/default.nix:10]:
```suggestion
    rev = version;
```

**Comment by vincentbernat** [comment]
❦ 27 août 2020 11:16 -07, Lassulus:

> +    rev = "${version}";
>
> ```suggestion
>     rev = version;
> ```

Thanks. Dunno why I didn't do that!

**Pull request merged**